tests/suites/test_suite_psa_crypto.function

```diff
@@ -1,15 +1,11 @@
 /* BEGIN_HEADER */
 #include <stdint.h>
 
-#if defined(MBEDTLS_PSA_CRYPTO_SPM)
-#include "spm/psa_defs.h"
-#endif
-
 #include "mbedtls/asn1.h"
 #include "mbedtls/asn1write.h"
 #include "mbedtls/oid.h"
 
-#include "psa/crypto.h"
+#include "psa_helpers.function"
```

Comment:
I'm not sure if including such files is clean. If it's meant to be included, I'd recommend changing the extension to .h instead. If it's to be used as an input for the Python scripts, .function is fine.

Reply:
I'm open to renaming this file to `.inc` or `.c`. It isn't used directly by anything that isn't in this PR yet so it's still very cheap to change.

There is a precedent for a file that is only meant to be #included: `programs/psa/psa_constant_names_generated.c`. AFAIR nobody remarked on the extension at that time. So go with `.c` here?

Reply:
Since it's cheap, and there's a precedent, I'd go for it!

Reply:
`.c` is not so cheap because it means changing makefiles not do delete `*.c` on cleanup.

@AndrzejKurek I'm warming up to `.h`. The file contains function definitions, but they're still pretty small functions. `.h` isn't the right thing, but it's the easy thing.

Reply:
I've gone with `.h`.